tests/system/python/packages/test_lab.py

```diff
@@ -0,0 +1,548 @@
+                assert data[0]["reading"]["ema_longX"] != ""
+                break
+                # TODO: verify asset tracker entry
+
+            retries -= 1
+
+        if retries == 0:
+            assert "TIMEOUT! randomwalk and ema_longX data not seen in randomwalk graph." + foglamp_url + "/foglamp/asset/randomwalk?seconds=600"
+
+
+    def test_updated_randomwalk2_python35_filter(self, foglamp_url, retries, wait_time):
+        move_file = "mv scripts/trendc.py.bak scripts/trendc.py"
+        exit_code = os.system(move_file)
+        assert exit_code == 0
+
+        url = foglamp_url + '/foglamp/category/Random1_PF/script/upload'
+        script_path = 'script=@scripts/ema.py'
+        upload_script = "curl -sX POST '{}' -F '{}'".format(url, script_path)
+        exit_code = os.system(upload_script)
+        assert exit_code == 0
+
+        time.sleep(wait_time)
+
+        while retries:
+            get_url = "/foglamp/asset/randomwalk1?seconds=600"
+            data = get_request(foglamp_url, get_url)
+            if "randomwalk" in data[0]["reading"] and "ema" in data[0]["reading"]:
+                assert 0 < data[0]["reading"]["randomwalk"]
+                assert 0 < data[0]["reading"]["ema"]
+                # TODO: verify asset tracker entry
+                break
+            retries -= 1
+
+        if retries == 0:
+            assert "TIMEOUT! randomwalk and ema data not seen in randomwalk graph." + foglamp_url + "/foglamp/asset/randomwalk?seconds=600"
+
+
+@pytest.mark.skipif(os.uname()[4][:3] != 'arm', reason="only compatible with arm architecture")
+class TestEnviroPhat:
+    def test_enviro_phat(self, foglamp_url, retries, wait_time):
+        data = {"name": "Enviro", "type": "south", "plugin": "envirophat", "enabled": True,
+                "config": {"assetNamePrefix": {"value": "e_"}}}
+        post_request(foglamp_url, "/foglamp/service", data)
+
+        data = {"name": "Fahrenheit", "plugin": "expression",
+                "filter_config": {"name": "temp_fahr", "expression": "temperature*1.8+32", "enable": "true"}}
+        post_request(foglamp_url, "/foglamp/filter", data)
+
+        data = {"pipeline": ["Fahrenheit"]}
+        put_url = "/foglamp/filter/Enviro/pipeline?allow_duplicates=true&append_filter=true"
+        put_request(foglamp_url, put_url, data)
+
+        time.sleep(wait_time*2)
+
+        while retries:
+            get_url = "/foglamp/asset/e_weather?seconds=600"
+            data = get_request(foglamp_url, get_url)
+            if "temperature" in data[0]["reading"] and "max" in data[0]["reading"]:
+                assert data[0]["reading"]["temperature"] != ""
+                assert data[0]["reading"]["temp_fahr"] != ""
+                break
+            retries -= 1
+
+        if retries == 0:
+            assert "TIMEOUT! temperature and max data not seen in e_weather graph." + foglamp_url + "/foglamp/asset/e_weather?seconds=600"
+
+class TestEventEngine:
+    # Enable Event Engine
```

Comment:
put as docstring of test_event_engine if test name itself is not enough self explanatory

Reply:
remove the comment ` # Enable Event Engine` as you have put the docstring now